helm-charts/common/retriever-usvc/values.yaml

```diff
@@ -52,7 +61,7 @@ podSecurityContext: {}
   # fsGroup: 2000
 
 securityContext:
-  readOnlyRootFilesystem: true
+  readOnlyRootFilesystem: false
```

Comment:
Why rootfs cannot be read-only any more?

(If there's a reason, document it as a comment.)

Reply:
done